bridge/client/app/app.routing.ts

```diff
@@ -1,26 +1,37 @@
 import {NgModule} from '@angular/core';
-import {Routes, RouterModule} from '@angular/router';
+import {Routes, RouterModule, ExtraOptions} from '@angular/router';
 import {DashboardComponent} from './dashboard/dashboard.component';
 import {ProjectBoardComponent} from './project-board/project-board.component';
 import {EvaluationBoardComponent} from './evaluation-board/evaluation-board.component';
 import {ForwarderGuard} from './_forwarder/forwarder_guard';
+import {KtbUniformViewComponent} from './_views/ktb-uniform-view/ktb-uniform-view.component';
+import {KtbIntegrationViewComponent} from './_views/ktb-integration-view/ktb-integration-view.component';
+import {KtbSettingsViewComponent} from './_views/ktb-settings-view/ktb-settings-view.component';
+import {KtbServiceViewComponent} from './_views/ktb-service-view/ktb-service-view.component';
+import {KtbSequenceViewComponent} from './_views/ktb-sequence-view/ktb-sequence-view.component';
+import {KtbEnvironmentViewComponent} from './_views/ktb-environment-view/ktb-environment-view.component';
 
+const routingConfiguration: ExtraOptions = {
+  paramsInheritanceStrategy: 'always'
+};
 
 const routes: Routes = [
   {path: '', pathMatch: 'full', redirectTo: 'dashboard'},
   {path: 'dashboard', component: DashboardComponent},
-  {path: 'project/:projectName', component: ProjectBoardComponent},
-  {path: 'project/:projectName/uniform', component: ProjectBoardComponent},
-  {path: 'project/:projectName/integration', component: ProjectBoardComponent},
-  {path: 'project/:projectName/settings', component: ProjectBoardComponent},
-  {path: 'project/:projectName/service', component: ProjectBoardComponent},
-  {path: 'project/:projectName/service/:serviceName', component: ProjectBoardComponent},
-  {path: 'project/:projectName/service/:serviceName/context/:shkeptncontext', component: ProjectBoardComponent},
-  {path: 'project/:projectName/service/:serviceName/context/:shkeptncontext/stage/:stage', component: ProjectBoardComponent},
-  {path: 'project/:projectName/sequence', component: ProjectBoardComponent},
-  {path: 'project/:projectName/sequence/:shkeptncontext', component: ProjectBoardComponent},
-  {path: 'project/:projectName/sequence/:shkeptncontext/event/:eventId', component: ProjectBoardComponent},
-  {path: 'project/:projectName/sequence/:shkeptncontext/stage/:stage', component: ProjectBoardComponent},
+  {path: 'project/:projectName', component: ProjectBoardComponent, children: [
+      {path: '', pathMatch: 'full', component: KtbEnvironmentViewComponent},
```

Comment:
I would suggest to provide the environment view at `path: 'environment'` and use a redirect rule for `path: ''`

Reply:
After a discussion with @Kirdock, we leave it as it is for now, unless we introduce deep links for the environment view.

Reply:
I would change it when we really need it